front-end-ui/README.md

```diff
@@ -1,3 +1,143 @@
+
+![alt text](collateral/group.bmp)  
+
+We are done!  
+Now the system will automatically track of the occasion date and will automatically send a notification to the recipient on the day of the event. Note that this application only does occasion notifications for now.
+
+
+## Occasion execution
+
+If microservice-notification is up and running, the event will simply be logged for administrative purposes. 
+The log entry would look something like this:
+
+    Apr 15, xxxx x:xx:xx AM net.wasdev.sample.microprofile.notification.NotificationResource
+    INFO: Congratulations Stacy White! $100.00 has been contributed by Co-workers for Stacy's
+    Retirement Party.  Please select a gift from your wish list at http://my.wishlist.com/swhite.
+
+For visual purposes, you will see a notification on the screen. It will look like this:
+
+![alt text](collateral/groupOccasionNotificationLogged.bmp)
+ 
+Notice that the occasion has now disappeared from the view.  
+
+Now, using microprofile's Fault Tolerance, we can fallback to using another notification service.  
+We realize that notification service v1 does not really send notifications to recipients, so we have upgraded it to do so. V1_1 not only logs to a file as v1 did, but it will tweet to the Acme Gifts account that a notification was sent, and it will send direct message notifications to twitter users that follow acme gifts.  
+So, next, we will take down notification service v1 and we will start notification service v1.1 to test it without causing service interruptions. For that, we use microprofile's fault tolerance function. In particular we use the retry policy and fallback support.
+
+Back with our example, If you remember John created an occasion for Stacy's retirement party and after the occasion took place, the group now shows that a tweet was sent: 
+
+![alt text](collateral/groupOccasionNotificationTweeted.bmp)
+
+Now, lets look at the Acme Gift's twitter page. This is the twitter account acting on behalf of Acme Gifts.  
+
+Here you see that a tweet was sent.
+
+![alt text](collateral/groupOccasionNotificationAcmeGiftsTwitterAccountTweet.bmp)
+
+
+Acme Gifts also sent a direct message to the recipient whos is following Acme Gifts:
+
+![alt text](collateral/groupOccasionNotificationAcmeGiftsTwitterDirectMsgSent.bmp)
+
+
+The recipient, in turn, got the tweet from Acme Gifts:
+
+![alt text](collateral/groupOccasionNotificationAcmeGiftsTwitterDirectMsgReceived.bmp)
+
+
+We are done!
+
+
+## Optional Actions
+
+In the group view you can click on the overflow menu icon next to each occasion. There you will have 3 options that allow you to delete any occasion, edit any occasion, or run any occasion immediately.
+
+![alt text](collateral/groupOccasionOverflowMenu.bmp)
+
+
+If you click on the menu icon at the rop of the page and next to the username, you will see two opions, which will allow you to logout and edit your profile.
+
+
+![alt text](collateral/headerMenu.bmp)
```

Comment:
question - should we, somewhere in here, talk about what's in the frontend component, in addition to talking about how to use the application?